src/actions/geek-lab/sample.js

```diff
@@ -0,0 +1,24 @@
+exports.command = 'sample';
```

Comment:
thats the yargs structure we can leverage, way cleaner then adding a function and calling it inside of bin :)